learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -203,7 +219,7 @@ Not every *Node* has *Sequence Ports*. In fact, most do not.
 *Data Ports* ports contain typed values. Types can be any regular Godot types, 
 such as a boolean, an integer, a string, a Vector3, an array, any Object or Scene Node, etc. 
 A *Data Port* on the right side of a node is considered an output, while, 
-a port on the left side is an input. Connecting them allows to transfer information from a node to the next. 
+a port on the left side is an input. Connecting them allows information to transfer to the next node. 
```

Comment:
Rather than "transfer" I'd say "allows information to flow to the next node". It's more descriptive.